QWeb/internal/table.py

```diff
@@ -377,8 +378,8 @@ def get_columns(self) -> list[WebElement]:
                         // If the table doesn't have a header, count the columns in the first body row
                         var firstBodyRow = tableRef.querySelector('tbody tr');
                         if (firstBodyRow) {
-                            headers = firstBodyRow.querySelectorAll('th');
                         }
+                            headers = firstBodyRow.querySelectorAll('th');
```

Comment:
I don't understand this.
Isn't the if block now empty and useless?

Reply:
hmm, I do not remember changing that one...?